app/Configs.php

```diff
@@ -259,13 +259,19 @@ public static function hasExiftool()
 
 		// value not yet set -> let's see if exiftool is available
 		if ($has_exiftool == 2) {
-			$path = exec('command -v exiftool');
-			if ($path == '') {
+			try {
+				$path = exec('command -v exiftool');
+				if ($path == '') {
+					self::set('has_exiftool', 0);
+					$has_exiftool = false;
+				} else {
+					self::set('has_exiftool', 1);
+					$has_exiftool = true;
+				}
+			} catch (Exception $e) {
 				self::set('has_exiftool', 0);
 				$has_exiftool = false;
-			} else {
-				self::set('has_exiftool', 1);
-				$has_exiftool = true;
+				Logs::warning(__METHOD__, __LINE__, 'exec is disabled, set has_exiftool to 0.');
```

Comment:
Maybe `has_exiftool set to 0` or `has_exiftool has been set to 0` to clarify that this isn't an instruction.